pkgs/development/lisp-modules-new/doc/nix-cl.md

```diff
@@ -0,0 +1,98 @@
+## Use cases
+
+This page lists some possible use cases for nix-cl.
+
+## Pinning down the exact commits of libraries
+
+Sometimes, a bug is fixed upstream but is not yet available in package
+repositories such as Quicklisp or Ultralisp. The users have to wait
```

Comment:
I would probably say «a fix is available» if you want to mention Ultralisp — «fixed upstream» sounds like putting the fix into the main branch of the repo, and Ultralisp usually picks up things literally in a few minutes…